Resources/Maps/lighthouse.yml

```diff
@@ -71008,10 +71037,14 @@ entities:
   - uid: 15295
     components:
     - type: Transform
+      anchored: False
```

Comment:
This one looks funny (see also: Physics)

Reply:
yep i removed all the duplicate pipes